qiskit_experiments/library/hamiltonian/heat_zx.py

```diff
@@ -0,0 +1,315 @@
+            q_1: ┤ Rα(π/2) ├─░─┤1      ├┤ Rx(-1.0*angle) ├┤ Rβ(π) ├─░─┤ γ ├┤M├
+                 └─────────┘ ░ └───────┘└────────────────┘└───────┘ ░ └───┘└╥┘
+            c: 1/═══════════════════════════════════════════════════════════╩═
+                                                                            0
+
+                             (xN)
+                    ┌───┐    ░ ┌───────┐                       ░
+            q_0: ───┤ X ├────░─┤0      ├───────────────────────░─────────
+                 ┌──┴───┴──┐ ░ │  heat │┌───────────┐┌───────┐ ░ ┌───┐┌─┐
+            q_1: ┤ Rα(π/2) ├─░─┤1      ├┤ Rx(angle) ├┤ Rβ(π) ├─░─┤ γ ├┤M├
+                 └─────────┘ ░ └───────┘└───────────┘└───────┘ ░ └───┘└╥┘
+            c: 1/══════════════════════════════════════════════════════╩═
+                                                                       0
+
+        ZX-HEAT experiments are performed with combination of two
+        error amplification experiments shown above, where :math:`\alpha, \beta, \gamma`
+        depend on the interrogated error axis, namely,
+        (``X``, ``X``, ``I``), (``Y``, ``Y``, ``I``), (``Y``, ``Z``, ``Rx(-pi/2)``)
+        for amplifying X, Y, Z axis, respectively.
+        The circuit in the middle is repeated by N times for the error amplification.
+
+        For example, we amplify the X error in the simplified ``heat`` gate Hamiltonian
+
+        .. math::
+
+            Ht = \frac{\Omega_{ZX}(t) ZX + \Delta_{IX}(t) IX}{2}.
+
+        From the BCH formula we can derive a unitary evolution of the Hamiltonian
+
+        .. math::
+
+            U = A_{II} II + A_{IX} IX + A_{ZX} ZX + A_{ZI} ZI.
+
+        Since we have a known control qubit state throughout the echo sequence,
+        we can compute a partial unitary on the target qubit, namely,
+        :math:`U_{j} = A_{Ij} I + A_{Xj} X` for the control qubit state :math:`|j\rangle`.
+        Here :math:`A_{Ij} =\cos \theta_j` and :math:`A_{Xj} =-i \sin \theta_j`.
+        This form is exactly identical to the unitary of :math:`R_X(\theta_j)` gate,
+        with :math:`\theta_0 =\Delta_{IX} + \Omega_{ZX}` and
+        :math:`\theta_1 =\Delta_{IX} - \Omega_{ZX}`,
+        given we calibrated the gate to have :math:`\Omega_{ZX} = \phi + \Delta_{ZX}`
+        so that :math:`\phi` corresponds to the experiment parameter ``angle``,
+        or the angle of the controlled rotation we want,
+        e.g. :math:`\phi = \pi/2` for the CNOT gate.
+        The total evolution during the echo sequence will be expressed by
+        :math:`R_X(\pi + \Delta_{ZX} \pm \Delta_{IX})` for the control qubit state
+        0 and 1, respectively.
+
+        In the echo circuit, the non-local ZX rotation by :math:`\phi` is undone by
+        applying :math:`R_X(\mp \phi)` with sign depending on the control qubit state,
+        thus only rotation error :math:`\Delta_{ZX}` from the target
+        angle :math:`\phi` is selectively amplified.
```

Comment:
```suggestion
        In the echo circuit, the non-local ZX rotation of angle :math:`\phi` is undone by
        the :math:`R_X(\mp \phi)` rotation whose sign depends on the control qubit state.
        Therefore, the rotation error :math:`\Delta_{ZX}` from the target
        angle :math:`\phi` is amplified.
```

Reply:
Reading the next sentence I see that we also amplify `IX` so I removed the `only`.